### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by iKrystyan**
In this milestone, OI added
1. Media queries for the desktop version of a minimum width of 992px.
2. Changed background images for the first section and a contact form.
3.  Navigation bar for the desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,101 +54,125 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                    </ul>
+                    <p class="paragraph">
+                        A daily selection of privately personalised reads; no accounts or no sign-ups required.
+                    </p>
+                    <ul class="langs">
+                        <li class="lang1 languages">html</li>
+                        <li class="lang2 languages">css</li>
+                        <li class="lang3 languages">javascript</li>
+                    </ul>
+                    <button type="button" class="seeProject">See Project</button>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] **GOOD JOB SO FAR** 👏. Please add a hover effect to buttons to make your buttons better look. You can see img below between default hover effect colors.

## Image of button states
image

File: index.html [hunk 2]
```diff
@@ -41,101 +54,125 @@ <h4 class="connect">LET'S CONNECT</h4>
...
-                <li class="lang2 languages">css</li>
-                <li class="lang3 languages">javascript</li>
+            <h4 class="connect true">LET'S CONNECT</h4>
+            <ul class="second socials true">
+                <li><a href="https://github.com/Christianib003"><i class="fab fa-github"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/christian-iradukunda-byiringiro-657598226"><i
+                            class="fab fa-linkedin-in"></i></a></li>
+                <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fab fa-instagram"></i></a></li>
             </ul>
```
Comment (rustam-tolipov) [inline index.html:173]: - [ ] Last issue in the desktop version is that `let's connect` button and social icons are centered. Please move them to the left side and with that, your desktop version will be completed 💯👍

#### You can see your website below
image

#### The figma design
image

File: index.html [hunk 3]
```diff
@@ -190,7 +227,7 @@ <h1 class="contact-title">Contact me</h1>
             <br>
             <input type="email" name="email" class="area email" placeholder="Email" required>
             <br>
-            <textarea name="message" class="message" maxlength="500" placeholder="Write your message here" required></textarea>
+            <textarea name="message" class="area message" maxlength="500" placeholder="Write your message here" required></textarea>
```
Comment (rustam-tolipov) [inline index.html:230]: - [ ] There is just one issue in your mobile version that I am sure you can fix it 💻. Please kindly increase the height of textarea which helps users send you more messages when they want to contact you 🤗🍀

#### Your design
image

#### Figma design
image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @Christianib003   , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -41,83 +54,124 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                        <li class="lang2 languages">css</li>
+                        <li class="lang3 languages">javascript</li>
+                    </ul>
+                    <button type="button" class="seeProject">See Project</button>
+                </div>
+                
+            </div>
+        </section>
+    </div>
+    <section class="About Myself">
```
Comment (Hamzaoutdoors) [inline index.html:158]: - [x] Great job working on changes requested by the previous reviewer, But unfortunately, you need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to layout this section `About section`  so they can look closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .

You can use `flex-direction: column;` in the CSS what do you think

## **Your project**
real

## **Design**
design

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hi @Christianib003, 

Great work on making the changes requested by a previous reviewer 👏 👏🏼 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢 

### Hi @Christianib003 👋🏻, 

You have done a tremendous job making the changes requested 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — COMMENTED** [review]
Hi Christian, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that was already approved, I saw your message that you are repeating the module however in that case you need to create a new repo and start the project from scratch, you can copy the code you already used from your old project but you can't send a merged PR for review again. alternatively, you can revert the project to the first commit and start all over again (although I don't recommend this option).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Amrhub** [comment]
Hi @Christianib003,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.


## Required Changes ♻️
_Check the comments under the review._


- [ ] [REQUIRED] I know you are repeating the week unfortunately the new Microverse system requires you to just make a new PR. you can't use a previous one. that's all you are missing for approval. It would be appreciated if you let your next reviewer about this situation probably linking this PR

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by iKrystyan** [comment]
Does it mean I will have to start a new project? @Amrhub

**Comment by Amrhub** [comment]
You can copy and paste but has to be a fresh new

**Pull request merged**